fmriprep/workflows/tests.py

```diff
@@ -0,0 +1,35 @@
+"""Utilities and mocks for testing and documentation building."""
+import os
+from contextlib import contextmanager
+from pathlib import Path
+from pkg_resources import resource_filename as pkgrf
+from toml import loads
+from tempfile import mkdtemp
+
+
+@contextmanager
+def mock_config():
```

Comment:
Yes, it's going to be tricky, but that would be the right thing. For now, since this is only utilized in generating documentation, it seems to work well enough.